docs/api-tutorials/spies.md

```diff
@@ -0,0 +1,55 @@
+Mocha does not come equipped with spies, though libraries like Sinon provide this behaviour if desired. The following is an example of Mocha utilizing Sinon and Should.js to test an EventEmitter:
```

Comment:
[Feature] These are links in the current page, which is good and useful.
Let's keep to that:

```suggestion
Mocha does not come equipped with spies, though libraries like [Sinon](https://github.com/cjohansen/sinon.js) provide this behaviour if desired. The following is an example of Mocha utilizing Sinon and [Should.js](https://github.com/shouldjs/should.js) to test an EventEmitter:
```